utils/retryexecutor.go

```diff
@@ -38,9 +39,10 @@ func (runner *RetryExecutor) Execute() error {
 		}
 
 		log.Warn(runner.getLogRetryMessage(i, err))
-		// Going to sleep for RetryInterval seconds
+		// Going to sleep for RetryInterval milliseconds
 		if runner.RetriesInterval > 0 && i < runner.MaxRetries {
-			time.Sleep(time.Second * time.Duration(runner.RetriesInterval))
+			log.Info("Waiting " + strconv.Itoa(runner.RetriesInterval) + "ms before trying again")
```

Comment:
Can be simplified to - 
```
log.Info("Waiting", runner.RetriesInterval, "ms before trying again")
```
Notice that using a comma between arguments also takes care fo the spacing between sections of log message.